README.md

```diff
@@ -28,14 +28,14 @@ This has many advantages:
 
 ## When should I use Typha?
 
-We recommend using Typha only if you're using the Kubernetes API Datastore and you have more than 50 
-Kubernetes nodes.  While Typha can be used with etcd, etcd v3 is optimised to handle many clients
+We recommend using Typha only if you're using the Kubernetes API Datastore.
```

Comment:
Just a nit but there are two `using` right next to each other in that sentence... maybe re-word that bit ?